draggablepanel/src/main/java/com/github/pedrovgs/DraggableView.java

```diff
@@ -260,7 +266,7 @@ public void minimize() {
    */
   public void closeToRight() {
     if (viewDragHelper.smoothSlideViewTo(dragView, transformer.getOriginalWidth(),
-        getHeight() - transformer.getMinHeightPlusMargin())) {
+            getHeight() - transformer.getMinHeightPlusMargin())) {
```

Comment:
I'll be formatting the code with the correct checkstyle along with the other changes.